egs/material/s5/local/chain/decode_test.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=/home/dpovey/libs
```

Comment:
I think you can move this to your .bashrc and change it to:

export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/dpovey/libs